packages/web/src/components/EventsList/utils.tsx

```diff
@@ -0,0 +1,20 @@
+import { colors } from '../../theme/colors'
+
```

Comment:
Let's do something like this; I didn't realize it was being used across files too... this file and the `EventList` should use the same values so it doesn't break if one changes.
```suggestion

export type EventStatus = 'IN PROGRESS' | 'PAST' | 'FUTURE';

```

Feel free to use an enum if you feel that would be more appropriate.